### PR TITLE
99-Login Email input should have a consistant invalid email error mes…

### LINKED ISSUE #99: Login Email input should have a consistant invalid email error message
#### Describe the bug
There is an error state but no error message when the input field is invalid but not active.
There is also no error message when the email is invalid but contains an `@` symbol.

#### What is the expected behavior?
The login email input should have an error message that says "Please enter a valid email" when the criteria for an email address is not met whether it's focused or not. 

#### What are the steps to reproduce?
1.  Clone the repo (`git clone https://github.com/brightlayer-ui/angular-workflows.git`).
2. CD into the project (`cd angular-workflows`)
3. Run the example project (`cd login-workflow && yarn start:example`)
4. Navigate to http://localhost:4200/auth/login
5. Type in the invalid email "asdfasd@dafasdf" and observe the lack of error.

#### Screenshots or links to minimum reproduction example
Screen Shot 2021-06-24 at 2 02 52 PM
Screen Shot 2021-06-24 at 2 04 09 PM

### PR DESCRIPTION
**Pull request by raghavenc5**
…sage in the login page issue is fixed

Fixes #99  .

#### Changes proposed in this Pull Request:
-  Changed the idFieldactive status to fixed this issue
- 
- 

#### Screenshots / Screen Recording (if applicable)
-
Screenshot 2023-04-11 at 11 57 42 AM
Screenshot 2023-04-11 at 11 58 03 AM

#### To Test:
-  yarn precommit

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Comment by JeffGreiner-eaton** [comment]
looks good. Just update change log and add a new section for v4.0.1 and move the fix under 4.0.1

**Comment by raghavenc5** [comment]
> looks good. Just update change log and add a new section for v4.0.1 and move the fix under 4.0.1

@JeffGreiner-eaton  Already updated the changelog and issue fixed is also under milestone v4.0.1. Pls check and let me know if any changes required.

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## v4.0.0 (November 22, 2022)
```
Comment (surajeaton) [inline login-workflow/CHANGELOG.md:3]: This needs to be changed to 4.0.1 (Unreleased)

Comment (raghavenc5) [reply login-workflow/CHANGELOG.md]: @surajeaton Updated the Changelog file

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**